### PR TITLE
service.md localised to Hindi

### PR DESCRIPTION
**Pull request by NitishKumar06**
Signed-off-by: NitishKumar06 <justnitish06@gmail.com>

Fix: #1304

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @NitishKumar06 
Please don't remove PR template and try to use the PR template so that reviewers or maintainers aware this PR fulfils the checklist.

File: content/hi/service.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: सेवा
+status: पूरा हुआ
```
Comment (Garima-Negi) [inline content/hi/service.md:3]: status should be completed:
`status: Completed`

File: content/hi/service.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: सेवा
```
Comment (Garima-Negi) [inline content/hi/service.md:2]: please add the english name of term in parenthesis:
e.g title: सर्वरलेस (Serverless)

File: content/hi/service.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+status: पूरा हुआ
+category: संकल्पना
+tags: ["वास्तुकला", "", ""]
+---
+
+कृपया ध्यान दें कि आईटी में सेवा के कई अर्थ होते हैं।
+इस परिभाषा में, हम अधिक पारंपरिक एक पर ध्यान केंद्रित करेंगे: माइक्रोसर्विस की तरह सेवा।
```
Comment (Garima-Negi) [inline content/hi/service.md:9]:
```suggestion
हम एक अधिक पारंपरिक अर्थ पर ध्यान केंद्रित करेंगे
```

File: content/hi/service.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+कैसे या भले ही सेवाएं माइक्रोसर्विसेज से भिन्न हों या नहीं, यह बारीक है और अलग-अलग लोगों की अलग-अलग राय हो सकती है।
```
Comment (Garima-Negi) [inline content/hi/service.md:10]:
```suggestion
यदि सेवाएँ माइक्रोसर्विसेज से भिन्न हैं और वे कैसे भिन्न हैं, यह बारीक है और अलग-अलग लोगों की अलग-अलग राय हो सकती है।
```

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Review by Garima-Negi — APPROVED** [review]

**Review by sayantani11 — APPROVED** [review]
nicely done!

**Comment by jayesh-srivastava** [comment]
@Garima-Negi Isn't this suppose to target dev-hi branch?

**Pull request merged**